docs/Lexicon.md

```diff
@@ -594,6 +594,17 @@ of `Optional<Int>`.) Sugared types preserve information about the form
 and use of the type even though the behavior usually does not change
 (except for things like access control). Contrast with [canonical type](#canonical-type).
 
+## TBD
+
+Text-based dynamic library files (TBDs) are a textual representation of
+the information in a dynamic library / shared library that is required
+by the static linker - basically a symbol list of the exported symbols.
```

Comment:
```suggestion
by the static linker.
```
There is no reason to assume that it is just the symbol list.  There is technically additional metadata there that is inferred AIUI.